### PR TITLE
[ZH] Update `Skinning\skin.ini` and `Skinning\osu!`

### PR DESCRIPTION
**Pull request by SamHou0**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Skinning/skin.ini/zh.md [hunk 1]
```diff
@@ -94,7 +89,7 @@ outdated_since: a5a4d44b66fbe6446aefb341d712d1623b1aac80
 **允许按类别自定义箭头的皮肤**
 
 - 添加了 `arrow-generic.png`、`arrow-warning.png` 和 `arrow-pause.png`。
-- 删除了 `play-warningarrow.png`。
+- 弃用了 `play-warningarrow.png` 以支持上述内容。
```
Comment (CloneWith) [inline wiki/Skinning/skin.ini/zh.md:92]:
```suggestion
- 弃用了 `play-warningarrow.png` 而使用上述元素。
```

有点怪，这里我想是将原有元素用新元素取代的意思

File: wiki/Skinning/osu!/zh.md [hunk 2]
```diff
@@ -762,5 +757,23 @@ outdated_translation: true
 - 必须自定义 `hit300.png`。
 - 此元素用于 `hit300`、`hit300g` 和 `hit300k`。
 
+## 滑条失误提示 （仅Lazer）
+
+在 [Lazer](/wiki/Client/Release_stream/Lazer) 版本的游戏中，添加了当滑条部分失误时显示的新提示。下面的传统皮肤元素可被用来制作这些失误提示。它们在 osu!stable 中没有效果<!-- TODO link somewhere -->。
```
Comment (CloneWith) [inline wiki/Skinning/osu!/zh.md:762]:
```suggestion
在 [Lazer](/wiki/Client/Release_stream/Lazer) 版本的游戏中，新增了漏掉滑条部分时的提示。可以使用下面的传统皮肤元素自定义这些提示。osu!stable 不使用它们<!-- TODO link somewhere -->。
```

- skin 在这里可译为“自定义”，也就是修改皮肤显示的样子
- no effect 可考虑意译为不使用（实际情况也似乎确实如此）
- *添加* 与 *新* 可合并为 *新增*
- *失误* 一般对应游戏 miss 掉的部分（即漏掉）

File: wiki/Skinning/osu!/zh.md [hunk 3]
```diff
@@ -762,5 +757,23 @@ outdated_translation: true
...
+| 版本 | 所有 |
+| 可以使用动画？ | ![是][true] |
+| 可以在谱面中自定义？ | ![是][true] |
+| 混合模式 | 普通模式 |
+| 定位点 | 中心 |
+| 推荐标准大小 | 16x16 |
+
+|  | 文件名 | 描述 |
+| :-: | :-- | :-- |
+| ![](img/sliderendmiss.png "Default sliderendmiss@2x.png") | `sliderendmiss.png` | 当错过 [滑条尾部](/wiki/Gameplay/Hit_object/Slider/Slidertail) 时展示。 |
```
Comment (CloneWith) [inline wiki/Skinning/osu!/zh.md:775]:
```suggestion
| ![](img/sliderendmiss.png "默认的 sliderendmiss@2x.png") | `sliderendmiss.png` | 当漏掉[滑条尾部](/wiki/Gameplay/Hit_object/Slider/Slidertail)时显示。 |
```
图片的描述文字也要翻译的（

File: wiki/Skinning/osu!/zh.md [hunk 4]
```diff
@@ -762,5 +757,23 @@ outdated_translation: true
...
+| ![](img/slidertickmiss.png "Default slidertickmiss@2x.png") | `slidertickmiss.png` | 当错过 [滑条点](/wiki/Gameplay/Hit_object/Slider/Slider_tick)时展示。 |
```
Comment (CloneWith) [inline wiki/Skinning/osu!/zh.md:776]:
```suggestion
| ![](img/slidertickmiss.png "默认的 slidertickmiss@2x.png") | `slidertickmiss.png` | 当漏掉[滑条点](/wiki/Gameplay/Hit_object/Slider/Slider_tick)时显示。 |
```

**Comment by CloneWith** [comment]
Still expecting @SamHou0 's thoughts on unresolved comments.

**Comment by SamHou0** [comment]
@CloneWith All the comments are resolved after applying suggestions.

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**